Sources/Action/AppKitExtensions/NSControl+Action.swift

```diff
@@ -0,0 +1,30 @@
+import Cocoa
+import RxSwift
+import RxCocoa
+
+public extension Reactive where Base: NSControl {
```

Comment:
Same here, could be 

```swift
#if os(iOS)
    import UIKit
    typealias OSControl = UIControl
#elseif os(macOS)
    import Cocoa
    import AppKit
    typealias OSControl = NSControl
#endif

extension Reactive where Base: OSControl {

}
```

Reply:
Somewhat agree on this one, the base control is relatively agnostic